src/js/player.js

```diff
@@ -1653,7 +1654,11 @@ class Player extends Component {
     // Loop through each playback technology in the options order
     for (let i = 0, j = this.options_.techOrder; i < j.length; i++) {
       let techName = toTitleCase(j[i]);
-      let tech = Component.getComponent(techName);
+      let tech = Tech.getTech(techName);
+
+      if (!tech) {
+        tech = Component.getComponent(techName);
```

Comment:
Yep, this is basically to support the now-deprecated feature of registering techs as components.